app/screens/settings/about/about.tsx

```diff
@@ -171,6 +184,22 @@ const About = ({componentId, config, license}: AboutProps) => {
         popTopScreen(componentId);
     });
 
+    const copyToClipboard = useCallback(
+        () => {
+            const appVersion = `${intl.formatMessage({id: 'settings.about.version', defaultMessage: 'App Version:'})} ${intl.formatMessage({id: 'settings.about.build', defaultMessage: '{version} (Build {number})'},
+                {version: DeviceInfo.getVersion(), number: DeviceInfo.getBuildNumber()})}`;
+            const server = `${intl.formatMessage({id: 'settings.about.server.version.desc', defaultMessage: 'Server Version:'})} ${intl.formatMessage({id: serverVersion.id, defaultMessage: serverVersion.defaultMessage}, serverVersion.values)}`;
+            const database = `${intl.formatMessage({id: 'settings.about.database', defaultMessage: 'Database:'})}  ${intl.formatMessage({id: 'settings.about.database.value', defaultMessage: `${config.SQLDriverName}`})}`;
```

Comment:
Also, for example in this one, the second part of the string is just the value of a variable, so this should be done just as `{id: 'settings.about.database', defaultMessage: 'Database: {driverName}'}` and pass the variable through the values argument.

Reply:
I am quite not sure how the value is passed.